com.chenyifaer.FaForever.yml

```diff
@@ -0,0 +1,99 @@
+id: com.chenyifaer.FaForever
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: faforever
+
+# These are the permissions that the application needs
+# Read more about finishing here: https://docs.flatpak.org/en/latest/manifests.html#finishing
+finish-args:
+  # Share IPC namespace with the host
+  - --share=ipc
+  # show windows using X11, if Wayland is not available, overrides x11 socket permission. 
+  - --socket=fallback-x11
+  # Show windows with Wayland
+  - --socket=wayland
+  # GPU acceleration if needed
+  - --device=dri
+  # Access the network
+  - --share=network
+  # Play sound with PulseAudio
+  - --socket=pulseaudio
+  # Access the XDG download directory
+  - --filesystem=xdg-download
+  # Access the XDG music directory
+  - --filesystem=xdg-music
+
+modules:
+  - name: webkit2gtk-4.0
+    sources:
+      - type: archive
+        url: https://webkitgtk.org/releases/webkitgtk-2.44.0.tar.xz
+        sha256: c66530e41ba59b1edba4ee89ef20b2188e273bed0497e95084729e3cfbe30c87
+        x-checker-data:
+          type: html
+          url: https://webkitgtk.org/releases/
+          version-pattern: LATEST-STABLE-(\d[\.\d]+\d)
+          url-template: https://webkitgtk.org/releases/webkitgtk-$version.tar.xz
+    buildsystem: cmake-ninja
+    config-opts:
+      # taken from https://gitlab.gnome.org/GNOME/gnome-build-meta/-/blob/5a241420c2ee85cc5080dd7cf7ba3049fd80fcf1/elements/world/webkit2gtk-4.0.bst
+      - -DPORT=GTK
+      - -DENABLE_DOCUMENTATION=OFF
+      - -DENABLE_WEBDRIVER=OFF
+      - -DUSE_GTK4=OFF
+      - -DUSE_LIBBACKTRACE=OFF
+      - -DUSE_SOUP2=ON
+    modules:
+      - shared-modules/libsoup/libsoup-2.4.json
+
+      - name: unifdef
+        no-autogen: true
+        make-install-args:
+          - prefix=${FLATPAK_DEST}
+        sources:
+          - type: archive
+            url: https://dotat.at/prog/unifdef/unifdef-2.12.tar.xz
+            sha256: 43ce0f02ecdcdc723b2475575563ddb192e988c886d368260bc0a63aee3ac400
+        cleanup:
+          - '*'
+
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+
+  # for some controller mods (see https://github.com/isXander/Controlify/issues/31)
+  - shared-modules/libusb/libusb.json
+
+  - name: faforever
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - ar -x *.deb
+      - tar -xf data.tar.gz
```

Comment:
```suggestion
      - bsdtar -Oxf fa-forever.deb data.tar.xz | bsdtar -xf -
```

Reply:
The command should be `bsdtar -Oxf fa-forever.deb data.tar.xz | bsdtar -xf -`. In addition, does the runtime include the `bsdtar` command?

Reply:
Yes it does snd that's what I wrote

Reply:
Ah the it is a gzip here fixed